docs/features/nullable-reference-types.md

```diff
@@ -61,9 +61,10 @@ Trivial/optimized cases:
     For completeness, we would also recognize a NullableAttribute with a single value 0 (rather than an array of 0s),
     but compiler will never emit an attribute like this. 
 
+NullableAttribute(1) should be placed on a type parameter definition that has a `notnull` constraint.
 NullableAttribute(1) should be placed on a type parameter definition that has a `class!` constraint.
 NullableAttribute(2) should be placed on a type parameter definition that has a `class?` constraint.
-NullableAttribute(2) should be placed on a type parameter definition that has no type constraints, `class`, `struct` and `unmanaged` constraints and
+NullableAttribute(2) should be placed on a type parameter definition that has no type constraints, `notnull`, `class`, `struct` and `unmanaged` constraints and
```

Comment:
>constraints [](start = 86, length = 11)

I had trouble parsing this sentence, so was confused. Consider removing the word "constraints" here ("... that has no type, `notnull`, `class` ... constraints"). Also typo on next line (equivalent).